roles/installer/templates/tower_postgres.yaml.j2

```diff
@@ -54,7 +54,7 @@ spec:
             - name: POSTGRES_HOST_AUTH_METHOD
               value: '{{ postgres_host_auth_method }}'
           ports:
-            - containerPort: 5432
+            - containerPort: {{ awx_postgres_port | default('5432')}}
```

Comment:
The quotes around this port var needed to be removed, and the default port was added as an extra precaution.